assets/src/setup/pages/template-mode/index.js

```diff
@@ -2,11 +2,28 @@
  * WordPress dependencies
```

Comment:
This screen to be covered in #4706 . Changes to this file are just for testing this PR.